build/dependencies.props

```diff
@@ -1,6 +1,6 @@
-<Project>
+﻿<Project>
   <PropertyGroup>
-    <AspNetCoreLabsVersion>0.3.0-*</AspNetCoreLabsVersion>
+    <AspNetIntegrationTestingVersion>0.4.0-*</AspNetIntegrationTestingVersion>
```

Comment:
Huh, I guess this is fine since this is not a package release but I still find it strange that we start bumping version numbers twice for the 2.0 milestone.